packages/shared/src/dashboards/utilization-card/utilization-item.tsx

```diff
@@ -92,6 +95,15 @@ export const UtilizationItem: React.FC<UtilizationItemProps> = React.memo(
 
       humanAvailable = humanizeValue(max - current).string;
     }
+    const humanizedLegend =
+      !!showHumanizedInLegend && current ? (
+        <ChartLegend
+          themeColor={ChartThemeColor.purple}
+          data={[{ name: description + ' : ' + humanizeValue(current).string }]}
+        />
+      ) : (
+        <ChartLegend />
+      );
```

Comment:
let PF decide the default behaviour by itself (they can decide to use something other than `ChartLegend` as well)
```suggestion
    let humanizedLegend:  JSX.Element;
      if (!!showHumanizedInLegend && current)
        humanizedLegend = <ChartLegend
          themeColor={ChartThemeColor.purple}
          data={[{ name: description + ' : ' + humanizeValue(current).string }]}
        />
```